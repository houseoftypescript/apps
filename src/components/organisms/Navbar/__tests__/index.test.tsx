import { render } from '@testing-library/react';
import { Navbar } from '../index';

describe('Navbar', () => {
  it('to match snapshot', () => {
    const { container } = render(<Navbar icon={<></>} appName={''}></Navbar>);
    expect(container).toMatchSnapshot();
  });
});
