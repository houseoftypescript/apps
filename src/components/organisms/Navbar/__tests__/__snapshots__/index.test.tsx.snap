// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar to match snapshot 1`] = `
<div>
  <nav
    class="py-8"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="flex justify-between text-white"
      >
        <h1
          class="text-2xl uppercase"
        >
          <a
            href="/"
          >
            <div
              class="flex items-center gap-2"
            />
          </a>
        </h1>
        <div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium p-0 text-white css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
